docs/source/conf.py

```diff
@@ -74,6 +73,23 @@
 # Prefix added to all the URLs generated in the 404 page.
 notfound_urls_prefix = ''
 
+# -- Options for markdown extension
+scylladb_markdown_enable = True
+scylladb_markdown_recommonmark_versions = [
+    'scylla-3.7.2.x',
+    'scylla-3.10.2.x',
+    'scylla-3.11.0.x',
+    'scylla-3.11.2.x',
+    'scylla-4.7.2.x',
+    'scylla-4.10.0.x',
+    'scylla-4.11.1.x',
+    'scylla-4.12.0.x',
+    'scylla-4.13.0.x',
+    'scylla-4.14.1.x',
+    'scylla-4.15.0.x',
```

Comment:
Why do we have so many of these?
I think we should keep only 2-3 for each major version (3.x and 4.x).
Also, how can we set that these will be updated automatically from the repository releases?
Today our latests are 4.17.0.0 and 3.11.5.3

Reply:
> Why do we have so many of these?

`scylladb_markdown_recommonmark_versions` lists versions using the old recommonmark library instead of the new MystParser. You don't need to change this if you add new versions to `BRANCHES` in `conf.py`.



> I think we should keep only 2-3 for each major version (3.x and 4.x).

[I agree](https://sphinx-theme.scylladb.com/stable/configuration/multiversion.html#listing-new-versions).  Maintainers can choose which branches to build docs for using the  `BRANCHES`  property in `conf.py`.

> Also, how can we set that these will be updated automatically from the repository releases?

There's an open issue for it: https://github.com/scylladb/sphinx-scylladb-theme/issues/848